### ISSUE TITLE
Using debugger, geany crashes when I hit the "run" button

### ISSUE THREAD

**Issue by andy5995** [geany/geany-plugins#491]
On Debian 8 (amd64)

libgtk-2.24.25-3
libglib-2.42.1-1

Here's the [terminal output](https://gist.github.com/andy5995/3ac8cd6dcdbb7c7fd0a13321e7fab6e0)

I have a .c file opened in geany. I click on the `debug` tab in the lower left corner. Over on the right, I click the arrow to run. Geany crashes.

I set a breakpoint in the file. Do everything mentioned above. Geany crashes.

Both geany and the plugin were installed from the Debian 8 repo.

Should I build geany and the plugin from source? Install from backports? Any suggestions how I might get the debugger working?

**Comment by andreygursky** [comment]
Do you mean the "debugger" plugin or "scope"? The first one does work for me, the latter one makes geany hanging (https://github.com/geany/geany-plugins/issues/433).

**Comment by scienception** [comment]
I've got the same problem. I'm working on a Mac. Did you manage to solve the problem?

**Comment by andy5995** [comment]
Funny that you mention it...

I hadn't done much coding for a while.. a week ago I tried this plug-in again. As I recall, geany froze (or crashed) when I hit step.

I tried again after reading your comment. I couldn't reproduce the crash.

I'm using Debian 9.

@andreygursky "Debugger", gdb selected.

I'll use the plug-in more often now in the future. If I can reproduce the problem, I'll be sure to document right away.

Thanks again for geany.
